src/main/services/open-web-torrent.ts

```diff
@@ -0,0 +1,45 @@
+import axios, { AxiosResponse } from "axios";
+import { Agent } from "https";
+
+type OpenWebTorrentResponse =
+  | {
+      seeds: number;
+      peers: number;
+    }
+  | {
+      error: {
+        code: number;
+        message: string;
+      };
+    };
+
+const httpsAgent = new Agent({
+  rejectUnauthorized: false,
+});
+const axiosRef = axios.create({
```

Comment:
You don't need a whole axios instance for a single request;
Plus, you should use https://www.npmjs.com/package/webtorrent-health instead (it's the same library used by the checker).

Reply:
I didn't know about that, thanks!